tensorflow_datasets/image/deeplesion.py

```diff
@@ -0,0 +1,206 @@
+    'https://nihcc.box.com/shared/static/2dn2kipkkya5zuusll4jlyil3cqzboyk.zip',
+    'https://nihcc.box.com/shared/static/peva7rpx9lww6zgpd0n8olpo3b2n05ft.zip',
+    'https://nihcc.box.com/shared/static/2fda8akx3r3mhkts4v6mg3si7dipr7rg.zip',
+    'https://nihcc.box.com/shared/static/ijd3kwljgpgynfwj0vhj5j5aurzjpwxp.zip',
+    'https://nihcc.box.com/shared/static/nc6rwjixplkc5cx983mng9mwe99j8oa2.zip',
+
+    'https://nihcc.box.com/shared/static/rhnfkwctdcb6y92gn7u98pept6qjfaud.zip',
+    'https://nihcc.box.com/shared/static/7315e79xqm72osa4869oqkb2o0wayz6k.zip',
+    'https://nihcc.box.com/shared/static/4nbwf4j9ejhm2ozv8mz3x9jcji6knhhk.zip',
+    'https://nihcc.box.com/shared/static/1lhhx2uc7w14bt70de0bzcja199k62vn.zip',
+    'https://nihcc.box.com/shared/static/guho09wmfnlpmg64npz78m4jg5oxqnbo.zip',
+    'https://nihcc.box.com/shared/static/epu016ga5dh01s9ynlbioyjbi2dua02x.zip',
+    'https://nihcc.box.com/shared/static/b4ebv95vpr55jqghf6bthg92vktocdkg.zip',
+    'https://nihcc.box.com/shared/static/byl9pk2y727wpvk0pju4ls4oomz9du6t.zip',
+    'https://nihcc.box.com/shared/static/kisfbpualo24dhby243nuyfr8bszkqg1.zip',
+    'https://nihcc.box.com/shared/static/rs1s5ouk4l3icu1n6vyf63r2uhmnv6wz.zip',
+
+    'https://nihcc.box.com/shared/static/7tvrneuqt4eq4q1d7lj0fnafn15hu9oj.zip',
+    'https://nihcc.box.com/shared/static/gjo530t0dgeci3hizcfdvubr2n3mzmtu.zip',
+    'https://nihcc.box.com/shared/static/7x4pvrdu0lhazj83sdee7nr0zj0s1t0v.zip',
+    'https://nihcc.box.com/shared/static/z7s2zzdtxe696rlo16cqf5pxahpl8dup.zip',
+    'https://nihcc.box.com/shared/static/shr998yp51gf2y5jj7jqxz2ht8lcbril.zip',
+    'https://nihcc.box.com/shared/static/kqg4peb9j53ljhrxe3l3zrj4ac6xogif.zip'
+]
+
+_CSV_URL = "https://raw.githubusercontent.com/anir16293/Deep-Lesion/master/DL_info.csv"
+# Path to images and category labels in data dir
+_DATA_DIR = "Deeplesion-v0.1.0"
+_TRAIN_LABELS_FNAME = "train.csv"
+_VALIDATION_LABELS_FNAME = "valid.csv"
+
+_ANNOTATION_CATEGORY = collections.OrderedDict({
+    '1':'Bone', 
+    '2':'Abdomen', 
+    '3':'Mediastinum', 
+    '4':'Liver', 
+    '5':'Lung', 
+    '6':'Kidney', 
+    '7':'Soft tissue', 
+    '8':'Pelvis', 
+    '-1':'unknown',
+})
+
+
+class Deeplesion(tfds.core.GeneratorBasedBuilder):
+  """DeepLesion dataset."""
+
+  VERSION = tfds.core.Version('0.1.0')
+
+  def _info(self):
+    return tfds.core.DatasetInfo(
+        builder=self,
+        description=_DESCRIPTION,
+        features=tfds.features.FeaturesDict({
+            "name":tfds.features.Text(),
+            "image":tfds.features.Image(encoding_format='png'),
+            "bbox":tfds.features.BBoxFeature(),
+            "label":tfds.features.ClassLabel(names=_ANNOTATION_CATEGORY.values()),
+        }),
+        # Homepage of the dataset for documentation
+        urls=[_URL],
+        citation=_CITATION,
+    )
+
+
+  def _split_generators(self, dl_manager):
+    """Returns SplitGenerators."""
+    path = dl_manager.manual_dir
```

Comment:
Can we make this an auto download?

Reply:
No problem, I'll add this feature as soon as possible.